File: SSNodeJsChat-Android/app/src/main/java/in/newdevpoint/ssnodejschat/stickyheader/stickyData/HeaderData.java
package in.newdevpoint.ssnodejschat.stickyheader.stickyData;


import androidx.annotation.LayoutRes;

import in.newdevpoint.ssnodejschat.model.StickyMainData;


public interface HeaderData extends StickyMainData {
	@LayoutRes
	int getHeaderLayout();

}

File: SSNodeJsChat-Android-Kotlin/app/src/main/java/in/newdevpoint/ssnodejschat/stickyheader/stickyData/HeaderData.kt
package `in`.newdevpoint.ssnodejschat.stickyheader.stickyData

import `in`.newdevpoint.ssnodejschat.model.StickyMainData
import androidx.annotation.LayoutRes

interface HeaderData : StickyMainData {
    @LayoutRes
    fun getHeaderLayout(): Int
}